colours/src/App.js

```diff
@@ -1,16 +1,20 @@
-import './App.css';
+import "./App.css";
```

Comment:
This file has a lot of linting changes, you may need to update your prettier config